lang/en-gb.xml

```diff
@@ -439,11 +438,11 @@ It is also generally recommended for GUI.</firmware_font_package_note>
 	<trophy_collection>
 		<search>Search</search>
 		<delete_trophy>Delete Trophy</delete_trophy>
-		<trophy_deleted>This trophy information saved on this user will be deleted.</trophy_deleted>
+		<trophy_deleted>Trophy information saved for this user will be deleted.</trophy_deleted>
```

Comment:
revert, official string is
"This trophy information saved on this system will be deleted."